ffi/capi/build.rs

```diff
@@ -0,0 +1,38 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use std::env;
+use std::path::PathBuf;
+
+/// Inform cargo of the include directories as metadata key value pairs
+///
+/// Cargo will make the values available to consumers via `DEP_ICU_CAPI_<KEY>`.
```

Comment:
It's `DEP_<links_key_in_uppercase>`, so `DEP_ICU_CAPI1_<key>` if we added the 1 to the links key (which might confuse other people too).